### PR TITLE
div p added

### PR DESCRIPTION
**Pull request by Tropik3572**
create method
created test

### CODE REVIEW

File: src/test/java/CalculatorTest.java [hunk 1]
```diff
@@ -23,6 +23,12 @@ public void sum() throws Exception {
         assertEquals(8, result);
     }
 
+    @Test
+    public void divP() throws Exception {
+        int result = Calculator.divP(5, 3);
+        assertEquals(2, result);
```
Comment (Tropik3572) [inline src/test/java/CalculatorTest.java]: cool

File: src/main/java/Calculator.java [hunk 2]
```diff
@@ -21,4 +21,8 @@ public static double div(double a, double b) {
         }
         return a / b;
     }
+
+    public static int divP(int a, int b) {
+        return  a % b;
```
Comment (Tropik3572) [inline src/main/java/Calculator.java]: shit

File: .travis.yml [hunk 3]
```diff
@@ -8,6 +8,6 @@ cache:
   directories:
     - $HOME/.m2
 install:
-    - mvn clean install -Dgpg.skip=true
```
Comment (Tropik3572) [inline .travis.yml]: wow

File: src/main/java/Calculator.java [hunk 4]
```diff
@@ -21,4 +21,8 @@ public static double div(double a, double b) {
         }
         return a / b;
```
Comment (afniko) [inline src/main/java/Calculator.java]: test review

File: src/main/java/Calculator.java [hunk 5]
```diff
@@ -21,4 +21,8 @@ public static double div(double a, double b) {
...
     }
+
+    public static int divP(int a, int b) {
```
Comment (afniko) [inline src/main/java/Calculator.java]: this don`t work

File: src/main/java/Calculator.java (hunk 2 above)
Comment (afniko) [reply src/main/java/Calculator.java]: +1

File: src/test/java/CalculatorTest.java (hunk 1 above)
Comment (rederep) [reply src/test/java/CalculatorTest.java]: nice

File: .travis.yml (hunk 3 above)
Comment (rederep) [reply .travis.yml]: iio

File: .idea/workspace.xml [hunk 6]
```diff
@@ -678,8 +653,14 @@
...
       <configuration name="CalculatorTest" configurationId="JUnit" />
     </history-entry>
-    <history-entry file="CalculatorTest_mulWithZero - 2018.06.29 at 21h 19m 28s.xml">
-      <configuration name="CalculatorTest.mulWithZero" configurationId="JUnit" />
+    <history-entry file="CalculatorTest_divP - 2018.07.02 at 20h 17m 06s.xml">
+      <configuration name="CalculatorTest.divP" configurationId="JUnit" />
+    </history-entry>
+    <history-entry file="CalculatorTest_divP - 2018.07.02 at 20h 17m 31s.xml">
+      <configuration name="CalculatorTest.divP" configurationId="JUnit" />
+    </history-entry>
```
Comment (rederep) [inline .idea/workspace.xml]: fvesaf fewef weqf

**Pull request merged**